main.go

```diff
@@ -509,6 +509,8 @@ ReadConfigAndRun:
 						logger.PrintError("Error: Reload requested, but ignoring since configuration errors are present")
 						exitCode = 1
 					}
+				} else if !success {
+					exitCode = 1
```

Comment:
would it be helpful to have some log here too?

Reply:
I don't think that's necessary, since the test command already outputs the failure information - in the case of a reload we added this because its helpful to know that the reload was skipped.